service/src/test/java/com/myhome/controllers/HouseControllerTest.java

```diff
@@ -194,17 +195,15 @@ void addHouseMembers() {
     // given
     Set<HouseMember> testMembers = TestUtils.HouseMemberHelpers.getTestHouseMembers(TEST_HOUSE_MEMBERS_COUNT);
     Set<HouseMemberDto> testMembersDto = testMembers.stream()
-        .map(member -> HouseMemberDto.builder()
+        .map(member -> new HouseMemberDto()
             .memberId(member.getMemberId())
-            .name(member.getName())
-            .build())
+            .name(member.getName()))
         .collect(Collectors.toSet());
 
-    AddHouseMemberRequest request = new AddHouseMemberRequest(testMembersDto);
+    AddHouseMemberRequest request = new AddHouseMemberRequest().members(testMembersDto);
 
-    Set<AddHouseMemberResponse.HouseMember> addedMembers = testMembers.stream()
-        .map(member -> new AddHouseMemberResponse.HouseMember(member.getMemberId(),
-            member.getName()))
+    Set<com.myhome.model.HouseMember> addedMembers = testMembers.stream()
+        .map(member -> new com.myhome.model.HouseMember().memberId(member.getMemberId()).name(member.getName()))
```

Comment:
Can we reformat it in a more readable way?

Reply:
Done!